paddle/fluid/operators/einsum_op.cc

```diff
@@ -33,6 +33,10 @@ class EinsumOpMaker : public framework::OpProtoAndCheckerMaker {
     AddInput("Operands", "(TensorList), The input tensor of einsum op.")
         .AsDuplicable();
     AddOutput("Out", "(Tensor), The output tensor of einsum op.");
+    AddOutput(
+        "Cache",
```

Comment:
It's not a good name for "Cache". Recommend to use more significant word such as inter_cache or tmp_out or other word